docs/source/contributing.rst

```diff
@@ -399,3 +402,30 @@ and ``scico-data`` repositories must be updated and kept in sync.
    ::
 
       git submodule foreach --recursive 'git push' && git push
+
+
+Building Documentation
+----------------------
+
+A local copy of the documentation can be built from the respository root directory by doing
+
+::
+
+  python setup.py build_sphinx
+
+
+Alternatively:
+
```

Comment:
It's not entirely "alternatively" because the docs requirements also have to be installed for the previous approach.

Reply:
Fixed it.